satpy/readers/nc_slstr.py

```diff
@@ -203,11 +203,10 @@ def get_dataset(self, key, info):
         variable = self.nc[self.datasets[key.name]]
 
         values = (np.ma.masked_equal(variable[:],
-                                     variable.attrs['_FillValue'], copy=False) *
+                                     variable.attrs.get('_FillValue', np.nan), copy=False) *
```

Comment:
Reading the nc file with xarray's function directly should take care of scaling and masking

Reply:
Thanks, that's a good point, I tried to avoid touching any more code then necessary, but it makes sense do this as well. I'll look into it.